addins/Cake.KeePass.yml

```diff
@@ -2,7 +2,7 @@ Name: Cake.KeePass
 NuGet: Cake.KeePass
 Assemblies:
 - "/**/Cake.KeePass.dll"
-Repository: https://github.com/Ashthos/Cake.KeePass
+Repository: https://github.com/ashthos/Cake.KeePass
```

Comment:
@Jericho https://github.com/Ashthos/Cake.KeePass seems to be the "correct" uri.

Reply:
This value was taken straight from the addin's nuspec. See the "Project Site" link in [this page](https://www.nuget.org/packages/Cake.KeePass/).